File: xcode-core/examples/go/simple_function.go
// Package main 简单函数 trace 示例
package main

import "fmt"

func greet(name string) string {
	return fmt.Sprintf("Hello, %s!", name)
}

func add(a, b int) int {
	return a + b
}

func main() {
	fmt.Println(greet("World"))
	fmt.Println(add(1, 2))
}

File: xcode-core/examples/rust/src/simple_function.rs
// 简单函数 trace 示例

fn greet(name: &str) -> String {
    format!("Hello, {}!", name)
}

fn add(a: i32, b: i32) -> i32 {
    a + b
}

fn main() {
    println!("{}", greet("World"));
    println!("{}", add(1, 2));
}
